Library/Homebrew/test/rubocops/livecheck/regex_if_page_match_spec.rb

```diff
@@ -0,0 +1,36 @@
+# typed: false
+# frozen_string_literal: true
+
+require "rubocops/livecheck"
+
+describe RuboCop::Cop::FormulaAudit::LivecheckRegexIfPageMatch do
+  subject(:cop) { described_class.new }
+
+  it "reports an offense when there is no `regex` for `strategy :page_match`" do
+    expect_offense(<<~RUBY)
+      class Foo < Formula
+        url "https://brew.sh/foo-1.0.tgz"
+
+        livecheck do
+        ^^^^^^^^^^^^ A `regex` is required if `strategy :page_match` is present.
+          url :stable
+          strategy :page_match
+        end
+      end
+    RUBY
+  end
+
+  it "rreports no offenses when a `regex` is specified for `strategy :page_match`" do
```

Comment:
```suggestion
  it "reports no offenses when a `regex` is specified for `strategy :page_match`" do
```
Was just checking the `livecheck`-related stuff here and spotted this, I believe it my be my fault 😅

Reply:
Good catch, thanks!